sympy/simplify/tests/test_simplify.py

```diff
@@ -478,6 +478,10 @@ def test_simplify_expr():
 
     assert simplify(hyper([], [], x)) == exp(x)
 
+    M = MatrixSymbol('M', 10, 10)
```

Comment:
Please put these in their own test, with a reference to this issue. At a check that involves `MatrixSymbol`s on their own (not a `MatrixElement`) should be added as well.